template/.github/workflows/build.yml.j2

```diff
@@ -16,6 +16,9 @@ on:
       - '[0-9][0-9].[0-9]+.[0-9]+'
   pull_request:
   merge_group:
+  schedule:
+    - cron: '15 3 * * 6'
+  workflow_dispatch:
```

Comment:
Below (line 90) is a script that checks why the run was triggered. Is this already adopted to work with the new trigger types?

Reply:
Good spotting:

https://github.com/stackabletech/operator-templating/blob/259470bab1b67b372048fdae5aaf2fb34754e8bc/template/.github/workflows/build.yml.j2#L96-L99

Reply:
Thinking about this a little more, maybe can we ignore publishing charts completely for these new triggers:
* on `schedule`: The purpose was to only rebuild the image to catch any security updates
* on `workflow_dispatch`: Manually triggering is more ambiguous, becuase the _users_ intention might not be for rebuilding the image. Would we want to publish helm charts when the GITHUB_REF is on the following?
    * for `refs/heads/main`, the intention is probably to rebuild an image earlier than the next schedule, so no need to publish charts.
    * for `refs/heads/*`, Unknown intent, probably better taken care of by a `pull_request` and the existing conditions.
    * for `refs/tags/*`, probably should not be run? IMO, these should be static for a release, so push a new tag instead (unless we nede to consider non-release tags too.

Regarding image builds, would we want to avoid build/publish on `workflow_dispatch` for `refs/tags/*`?

Reply:
Marking this as resolved, as I believe future changes cover these items.